.github/workflows/all.yml

```diff
@@ -43,27 +43,17 @@ jobs:
 
   lint:
     runs-on: ubuntu-latest
-    env:
-        GO111MODULE: 'on'
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - name: Install Go
-        uses: actions/setup-go@v2
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
         with:
-          go-version: 1.17.x
-      - name: Install golangci-lint
-        working-directory: /tmp
-        run: |
-          GOLANGCI_LINT_VERSION=$(head -n 1 "${GITHUB_WORKSPACE}/.golangci.yml" | tr -d '# ')
-          go install "github.com/golangci/golangci-lint/cmd/golangci-lint@$GOLANGCI_LINT_VERSION"
-      - name: Run linters
-        run: |
-          BASEREV=$(git merge-base HEAD origin/master)
-          echo "Base revision: $BASEREV"
-          golangci-lint run --out-format=tab --new-from-rev "$BASEREV" ./...
+          # Required: the version of golangci-lint is required and must be specified without patch version: we always use the latest patch version.
+          version: v1.43
```

Comment:
Awesome stuff :+1: 

But we still can keep the version in a single place using `::set-output` (https://stackoverflow.com/a/65616499)

Reply:
done in [4119c40](https://github.com/grafana/k6/pull/2397/commits/4119c40d75147a5b314ae07bd935e089037b926d)